integration-tests/src/test/java/com/datastax/oss/driver/mapper/InventoryITBase.java

```diff
@@ -85,6 +86,13 @@
   }
 
   private static final Version MINIMUM_SASI_VERSION = Version.parse("3.4.0");
+  private static final Version BROKEN_SASI_VERSION = Version.parse("6.8.0");
+
+  protected static boolean isSasiBroken(CcmRule ccmRule) {
+    Optional<Version> dseVersion = ccmRule.getDseVersion();
+    // creating SASI indexes is broken in DSE 6.8.0
```

Comment:
shouldn't we report it somewhere? Is there a JIRA issue that captures that?

Reply:
I opened [DSP-20720](https://datastax.jira.com/browse/DSP-20720) against this. It's mentioned in the PR description above too. :wink: